docs/index.md

```diff
@@ -38,6 +44,12 @@ The `Arcus.WebApi.Logging` package contains functionality that can be incorporat
 
 - [Logging unhandled exceptions](features/logging.md)
 
+## Arcus.WebApi.OpenApi.Extensions
+
+The `Arcus.WebApi.OpenApi.Extensions` package contains functionality that can be used to easily improve the Open API documentation of an API when making using of Swashbuckle.
+
+- [Exposing security information in Swashbuckle documentation](features/openapi.securitydefinitions.md)
```

Comment:
Let's use a dash here for the sake of the URL
```suggestion
- [Exposing security information in Swashbuckle documentation](features/openapi-securitydefinitions.md)
```

It feels a bit odd, but that's my opinion:
![image](https://user-images.githubusercontent.com/4345663/60157953-30e88200-97f0-11e9-959f-79295de1822a.png)

Maybe `features/openapi/extensions` or `features/openapi/security-definitions` is even better?